core/templates/dev/head/pages/header_js_libs.html

```diff
@@ -13,3 +13,17 @@
 <script src="/third_party/static/headroom-js-0.9.4/headroom.min.js"></script>
 <script src="/third_party/static/headroom-js-0.9.4/angular.headroom.min.js"></script>
 <script src="/third_party/static/angular-drag-and-drop-lists-2.1.0/angular-drag-and-drop-lists.min.js"></script>
+
+<script>
```

Comment:
I thought: we were going to add a guard for whether the correct constant value is set, and that you were thinking about refactoring this into the siteAnalyticsService.

Also I liked how you encapsulated this in an IIFE here: https://github.com/oppia/oppia/pull/6835/files#diff-ad2ef59507114f6004c06bdc2ad92da4R23 -- maybe keep that?

Reply:
Yes, I thought of that, then I felt leaving it here will be better.
Okay, I'll continue with the IIFE. But I noticed that there isn't any much change since the `ga` variable is bound to the global window object